proto/v1beta1/intoto.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package grafeas.v1beta1.intoto;
+
+option go_package = "github.com/grafeas/grafeas/proto/v1beta1/intoto_go_proto";
+option java_multiple_files = true;
+option java_package = "io.grafeas.v1beta1.intoto";
+option objc_class_prefix = "GRA";
+
+import "google/protobuf/struct.proto";
+
+// This file defines the proto that matches the schema of an in-toto link
+// file. in-toto is a framework to protect a software supply chain. You can read
+// more about in-toto at https://in-toto.io/ and you can find the current
+// version of the in-toto specification at
+// https://github.com/in-toto/docs/blob/master/in-toto-spec.md
+
+
+// A signature object consists of the KeyID used and the signature itself.
+message Signature {
+  string keyid = 1 [json_name="keyid"];
```

Comment:
Remind me: is `json_name` to be able to convert into in-toto format more easily?

Reply:
`json_name` in this instance is able to easily map to the in-toto naming of the fields.

Reply:
Gotcha, is the intention for these fields to be identically named though? It seems that if the field in proto has the same name as in json, `json_name` is unnecessary.

Also, it's more idiomatic in protos to separate the words in snake case, eg `key_id`.